Library/Homebrew/cmd/cat.rb

```diff
@@ -25,6 +26,17 @@ def cat
     raise "`brew cat` doesn't support multiple arguments" if args.remaining.size > 1
 
     cd HOMEBREW_REPOSITORY
-    safe_system "cat", formulae.first.path, *Homebrew.args.passthrough
+    pager = "cat"
+    unless ENV["HOMEBREW_BAT"].nil?
```

Comment:
`HOMEBREW_BAT` would be OK.

I wish Homebrew had a convention of `HOMEBREW_USE_X` for this kind of thing.